rdtho2525/whats-cookin
# Feature/design updates

## Is this a fix or a feature? ⬇️

Feature

## What is the change? ⬇️

Incremental updates and tweaks to the design. Also, center piece was updated to use image as a background.

## Where should the reviewer start? ⬇️

Index.html and styles.css should be reviewed in the GitHub split view to verify all updates.

## How should this be tested? ⬇️

Open index.html in the browser and ensure that all updates bring the design closer to the spec and wireframe.
